### PR TITLE
Fix change due date

### PR DESCRIPTION
**Pull request by trowik**
a timezone had to be specified if you wanted to change the due date

### CODE REVIEW

**Review by winged — CHANGES_REQUESTED** [review]
Good work! However, instead of enforcing an exact date format, why not use a date parser that can handle multiple formats? Here's what I'm thinking about, shouldn't be too hard to implement:

https://dateparser.readthedocs.io/en/latest/
If we start using this library, we might even get rid of handling "3d", "4w" etc separately from a specific date. I've just tested a few strings, here's what I got (the settings part is important, as it defaults to preferring past dates otherwise):

```
>>> dateparser.parse('3d', settings={'PREFER_DATES_FROM': 'future'})
datetime.datetime(2020, 5, 23, 16, 19, 59, 730620)
>>> dateparser.parse('4w', settings={'PREFER_DATES_FROM': 'future'})
datetime.datetime(2020, 6, 17, 16, 20, 8, 687007)
>>> dateparser.parse('tuesday', settings={'PREFER_DATES_FROM': 'future'})
datetime.datetime(2020, 5, 26, 0, 0)
>>> dateparser.parse('friday', settings={'PREFER_DATES_FROM': 'future'})
datetime.datetime(2020, 5, 22, 0, 0)
>>> dateparser.parse('2020-10-03', settings={'PREFER_DATES_FROM': 'future'})
datetime.datetime(2020, 10, 3, 0, 0)
```

File: app/mails/views.py [hunk 1]
```diff
@@ -195,7 +196,9 @@ def mail_update_view(request):
             try:
                 due_id = int(re.sub(r"due-", "", due[0]))
                 d = Due.objects.get(mail=mail, pk=due_id)
-                d.due = due[1][0]
+                d.due = dateparser.parse(
```
Comment (winged) [inline app/mails/views.py:199]: You'll need to use the date parser for new dues as well (see a few lines above)

**Review by winged — CHANGES_REQUESTED** [review]

File: app/mails/views.py (hunk 1 above)
Comment (trowik) [reply app/mails/views.py:202]: done

Comment (trowik) [reply app/mails/views.py:202]: done

**Review by winged — APPROVED** [review]
Nice work!

**Pull request merged**